### ISSUE TITLE
[BUG] PIPELINE_LOAD_ERROR: An HTTP status code indicating failure was received: 403

### ISSUE THREAD

**Issue by wes1993** [mauriceoegerli/viewtube-vue#2989]
### Describe the Bug

When starting playing a video the error "PIPELINE_LOAD_ERROR: An HTTP status code indicating failure was received: 403" appear after some secons

### Steps to Reproduce the Bug

1. Chose Video
2. Start Playing

### Expected Behaviour

Video playing correctly

### Device Info

Device: PC
OS: Windows 11
Browser: Chrome
Version: 129.0.6668.90

**Comment by b1ck0** [comment]
yep same problem here when running in a docker container, tried the lastest and dev tags - same story

**Comment by liviuq** [comment]
same here.

**Comment by Lukeomichael** [comment]
I've just set up a new Docker and installed ViewTube.  All looking good until I start playing any video, and after a few seconds playing, I also get:

_Video playback error
PIPELINE_LOAD_ERROR: An HTTP status code indicating failure was received: 403

Within inspector:

    return n instanceof ae ? new Qn("PIPELINE_LOAD_ERROR",n) : Oe(n, {
        defaultCode: "PIPELINE_LOAD_ERROR",
        defaultReason: "Unknown error when fetching the Manifest"

**Comment by bmdumitrascu** [comment]
Same happens with manual install.

**Comment by DerSpatz** [comment]
Same here, latest version on Docker.

**Comment by Zetto12** [comment]
Any update on this? I have the same problem. Video play for 10 or so seconds and than I get a pipeline error

**Comment by LordS4me** [comment]
Any news on this ?

**Comment by JanJK2011** [comment]
I am wondering if this project is still alive. Because with this bug it's not usable for months now. Anyone got the scoop on its status?

**Comment by Kingo71** [comment]
> I am wondering if this project is still alive. Because with this bug it's not usable for months now. Anyone got the scoop on its status?

Wondering the same, any contributor willing to give an update?

**Comment by wouldntyouknow** [comment]
I think this has been abandoned due to this issue.

**Comment by ewoks** [comment]
Just to make it clear - the bug is already known (see #2952) and project is semi-paused too (#3002).

**_Let's please respect the maintainer and his civil service time, without bringing even more stress_**

previously consolidated notes:
- everyone affected is running it in a docker container;
- even immediately after new start, no video could be played
- issue is independent of container running standalone or behind a reverse proxy like traefik (I just tested both variants and can confirm the issue)

in the other news, it seems like `viewtube.io` is affected as well as like some users mention it doesn't even starts.

p.s. one or several more comments will not speed things up, rather slow them down.

Finally this can be closed as duplicate of #2952
